server/build.gradle

```diff
@@ -29,25 +28,17 @@ ext {
 }
 
 configurations {
-    // A configuration meant for consumers that need the API of this component
-    implementation {
-        canBeResolved = true
-    }
```

Comment:
We don't iterate over implementation artifacts directly, so we don't need it here (it's not supported in new Gradle, so it's easier to just remove it)